test/config.custom.yaml

```diff
@@ -36,3 +36,10 @@ electricity:
 
   extendable_carriers:
     Link: [H2 pipeline]
+
+# clean_cooking configuration
+clean_cooking:
+  enable: true
+  fuel: ["AC", "lpg", "pellets", "firewood", "charcoal", "heat"]
+  p_nom: [1, 1, 1, 1, 1, 1] # same order as fuels
+  p_max_pu: [1, 1, 1, 1, 1, 1] # same order as fuels
```

Comment:
Could you explain why you want to add here p_max_pu. What's the purpose for reducing it here? When people want that?